packages/xml-views-validation/api.d.ts

```diff
@@ -0,0 +1,26 @@
+import { XMLDocument } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+
+export function validateXMLView(opts: {
+  model: UI5SemanticModel;
+  xmlView: XMLDocument;
+}): UI5XMLViewIssue[];
+
+export interface UI5XMLViewIssueStructure {
```

Comment:
Maybe rename this to `BaseUI5XMLViewIssue`

Reply:
o.k.